engine/tests/phpunit/Elgg/TestCase.php

```diff
@@ -7,6 +7,10 @@
 use stdClass;
 use Zend\Mail\Transport\InMemory as InMemoryTransport;
 
+/**
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
```

Comment:
This has to be declared for each test class, it doesn't inherit. I confirmed by redefining a global function in 2 separate tests. Add these declarations and it's OK.